ocs_ci/utility/utils.py

```diff
@@ -4776,3 +4776,23 @@ def get_role_arn_from_sub():
         return role_arn
     else:
         raise ClusterNotInSTSModeException
+
+
+def get_os_version():
```

Comment:
We should probably update the logic to support at least Fedora/CentOS/RHEL .

```
pbalogh ~ $ podman run -it fedora  /usr/bin/bash
[root@b17935503af5 /]# cat /etc/redhat-release
Fedora release 40 (Forty)
```

Since Fedora 34 it's like RHEL 9. So we can probably go with some function based on fedora 

But our main problem is I guess glibc 2.34 is required so I think we should rather have a function which will check the version of glibc:

```
[root@66b3f15085e8 /]# ldd --version ldd
ldd (GNU libc) 2.34
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
[root@66b3f15085e8 /]# cat /etc/redhat-release
Red Hat Enterprise Linux release 9.4 (Plow)
```